ConfigureFreedns.sh

```diff
@@ -152,13 +161,6 @@ EOF
 # Start the first update immediately
 wget -O - --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 $directurl
 
-#Add the command to renew the script to the startup url
```

Comment:
Why was this part removed?
This would check weather this line was already added to rc.local and if not it would add it.
Are you sure that we will always add this part?

Reply:
Thanks for the review.

We now have a new script that takes care of the startup file (rc.local) both for a fresh install and an update.
This new file is this:
https://github.com/Navid200/cgm-remote-monitor/blob/FreeDNS_AutoLogin_PR/StartUpSetup.sh

Therefore, the task of the section you have identified it is taken care of by this new script.

I did this so that we have the startup modifications, in the future, in one place instead of in two different spots for fresh install versus for update.

Let's say someone who has an existing setup, after this PR, updates their platform.  After they do that, they will see a pink mark on their status page saying "FreeDNS ID and pass".
They will look at the status page key, where they will see that they need to run "Enter FreeDNS ID and password" from the Google Setup submenu.
Doing that will run the new script that enters the startup file content.